rtic-monotonics/src/stm32.rs

```diff
@@ -290,25 +290,17 @@ macro_rules! make_timer {
                     0
                 };
 
-                $timer.ccr(1).write(|r| r.set_ccr(val.into()));
+                $timer.ccr(0).write(|r| r.set_ccr(val.into()));
```

Comment:
Don't be this one, this is the user triggered interrupt I'm talking about. It gets set whenever the user says "wait 3 ms", so that an interrupt wakes up the program when 3ms was over.